### PR TITLE
Add gpu ci workflow

### PR DESCRIPTION
**Pull request by luzzyzhang**
- Add GPU workflow

### CODE REVIEW

**Comment by luzzyzhang** [comment]
暂时关掉了 `megdnn_test` 验证 megbrain_test, 修复版同步出来 rebase 后再放开 `megdnn_test`

**Comment by xiaocenxiaocen** [comment]
LGTM

**Comment by xxr3376** [comment]
Please squash your commit and place code change in a separate commit for easier cherry-pick.

**Comment by luzzyzhang** [comment]
> Please squash your commit and place code change in a separate commit for easier cherry-pick.
done

File: ci/run_cpp_test.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ export MGB_STABLE_RNG=1
 
 function cpp_test {
     pushd /tmp/build/"${1}" >/dev/null
-        ./dnn/test/megdnn_test
+        # ./dnn/test/megdnn_test
```
[the remark this answers is not in the document]
Comment (luzzyzhang) [reply ci/run_cpp_test.sh]: 修改了

**Pull request merged**